nix/nixos/cardano-cluster-service.nix

```diff
@@ -132,10 +132,13 @@ in let
 in let
   ### Config
   node-config-overlay   = {
-    hasEKG        = null;
-    hasGUI        = null;
-    hasGraylog    = null;
-    hasPrometheus = null;
+    hasEKG                  = null;
+    hasGUI                  = null;
+    hasGraylog              = null;
+    hasPrometheus           = null;
+    minSeverity             = "Debug";
+    TraceChainDb            = true;
+    TracingVerbosity        = "MaximalVerbosity";
```

Comment:
Right, maximal verbosity for this test. So it's ok to have the extra header trace detail only be included at maximal verbosity.